tools/clean/provider/gcp.go

```diff
@@ -19,18 +19,60 @@ type GCP struct {
 	projectID string
 
 	networkClient       *compute.NetworksClient
+	subnetworksClient   *compute.SubnetworksClient
 	forwardRuleClient   *compute.ForwardingRulesClient
 	addressClient       *compute.AddressesClient
 	keyManagementClient *kms.KeyManagementClient
 }
 
 func (gcp *GCP) DeleteVpc(ctx context.Context, vpcName string) error {
+	vpcGetRequest := &computepb.GetNetworkRequest{
```

Comment:
Does GCP require subnets to be remove VPCs with subnets? seems strange

Reply:
yes, it failed on my test until I added this code.